README.md

```diff
@@ -1,22 +1,30 @@
 [![PyPI version](https://badge.fury.io/py/logzio-python-handler.svg)](https://badge.fury.io/py/logzio-python-handler) [![Build Status](https://travis-ci.org/logzio/logzio-python-handler.svg?branch=master)](https://travis-ci.org/logzio/logzio-python-handler)
 
 # The Logz.io Python Handler
+
+<table><tr><th>
+
+### Deprecation announcement
+Version 2.1.0 of this project ends support for Python 2.7 & 3.4. We recommend migrating your projects to Python 3.5 or newer as soon as possible. We'll be happy to answer any questions you have in [a GitHub issue](https://github.com/logzio/logzio-python-handler/issues).
```

Comment:
Add a ends support for python 3.3 as well. I see we wrote something about it in the readme. And delete the section where we wrote we won't support python3.3

Reply:
done.